client/src/cli/apiToken.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  AccessTokenError,
+  getRmmbrAccessTokenPath,
+} from "./rmmbrAccessToken.ts";
+import { err, ok, Result } from "./deps.ts";
+
+const rmmbrServer = Deno.env.get("RMMBR_SERVER");
```

Comment:
don't use the service name

Reply:
What should I use? Just call it "server"?

Reply:
something like `serverURL`